middleware/cmd/middleware/main.go

```diff
@@ -28,8 +29,14 @@ func main() {
 	redisMock := flag.Bool("redismock", false, "Mock redis for development instead of connecting to a redis server, default is 'false', use 'true' as an argument to mock")
 	imageUpdateInfoURL := flag.String("updateinfourl", "https://shiftcrypto.ch/updates/base.json", "URL to query information about updates from (defaults to https://shiftcrypto.ch/updates/base.json)")
 	notificationNamedPipePath := flag.String("notificationNamedPipePath", "/tmp/middleware-notification.pipe", "notificationNamedPipe specifies the path where the Middleware creates a named pipe to receive notifications from other processes on the BitBoxBase (defaults to /tmp/middleware-notification.pipe)")
+	hsmSerialPort := flag.String("hsmserialport", "/dev/ttyUSB0", "The serial port to communicate with the HSM.")
```

Comment:
I guess this is the correct serial port for the RockPro64

Reply:
Nope, on the rockpro it will be `/dev/ttyS0` or maybe `/dev/ttyS2`.

Reply:
Then I'd prefer `/dev/ttyS0` here

Reply:
@Stadicus Do you remember which serial port is used on the rockpro?

Reply:
Pretty sure it was `/dev/ttyS0`

Reply:
fixed (force pushed)